app/extensions/brave/locales/en-US/menu.properties

```diff
@@ -89,6 +89,9 @@ bookmarksManager=Bookmarks…
 importBrowserData=Import Browser Data…
 exportBookmarks=Export Bookmarks…
 submitFeedback=Submit Feedback…
+braveHelpLaptop=Brave Help Laptop…
+braveHelpIOS=Brave Help IOS…
```

Comment:
@kumarrishav Could you please change this to `braveHelpIOS=Brave Help iOS…` (correct iOS capitalization)?